src/openms/source/FORMAT/AbsoluteQuantitationMethodFile.cpp

```diff
@@ -216,9 +210,58 @@ namespace OpenMS
     aqm.setTransformationModelParams(transformation_model_params);
   }
 
-  // void AbsoluteQuantitationMethodFile::store(const String & filename, const std::vector<AbsoluteQuantitationMethod> & aqm_list)
-  // {
-  //   // TODO: pending fix to CsvFile::fstore()
-  // }
-
+  void AbsoluteQuantitationMethodFile::store(
```

Comment:
Was there a reason that you chose not to use the Csv::addRow and Csv::store methods?

Reply:
I could convert each aqm to a StringList and then take advantage of those methods, yeah. Did not think about it. I'll try that solution as well and I'll see if there is anything wrong with it.